e2e/tests/session_steps_test.go

```diff
@@ -280,3 +279,33 @@ func (s *suite) waitForNewBlockEvent(targetEvent string) {
 		s.Log("Success; message detected before timeout.")
 	}
 }
+
+// abciToClaimSettledEvent converts an abci.Event to a tokenomics.EventClaimSettled
+func (s *suite) abciToClaimSettledEvent(event *abci.Event) *tokenomicstypes.EventClaimSettled {
+	var claimSettledEvent tokenomicstypes.EventClaimSettled
+	for _, attr := range event.Attributes {
+		switch string(attr.Key) {
```

Comment:
Is there no way to take advantage of the fact that we're using typed events and convert the `*abci.Event` more directly?

Reply:
See new TECHDEBT item

Reply:
:eyes: I'm looking but not seeing. Can you link me?

Reply:
![Screenshot 2024-06-14 at 9 34 07 AM](https://github.com/pokt-network/poktroll/assets/1892194/5484a625-0679-4c02-9705-9f36cbde5ad7)